test/include/test_runner/test_parser.h

```diff
@@ -0,0 +1,94 @@
+#include <cstring>
+
+#include "common/file_utils.h"
+#include "main/kuzu.h"
```

Comment:
Do we need to include `"main/kuzu.h"` ? Also `using namespace kuzu::main;`. Can you double check?